src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: External Resources
+description: Configure Okteto's manifest to include external resources in your development environment
+sidebar_label: External Resources
+id: external-resources
+---
+
+import Image from '@theme/Image';
+
+The External Resources feature in Okteto is a powerful mechanism that enables you to manage the lifecycle of external resources as part of your Okteto development environment. This means you can define the external resources that comprise your application, include those resources in Okteto's garbage collection process, and even destroy those resources when you finish working with your development environment. 
+
+Here are just a few examples of external resources you can use with Okteto:
+
+- A markdown file (e.g. README, API Documentation)
+- MongoDB Atlas Instance
+- LaunchDarkly
+- Amazon SQS
+- Amazon S3
+- Amazon RDS databases
+- AWS Lambda Functions
+
+You no longer need to use different tools to work on your application in Okteto and you won't need to worry about rogue external resources continuing to run once you are done with your work.
+
+When you configure External Resources in Okteto you can see those resources within the Okteto UI, can click to go directly to those resources in their respective platform, and include them in your Okteto `deploy` and `destroy` configurations.
+
+## Overview
+
+When you configure the `external` section in your [Okteto manifest](/reference/manifest), keep in mind the following:
+
+- You can create a container image that includes any external tools (to Okteto) that you need in order to provision or access your external resources.
+- This container image can be included in your Okteto manifest's `deploy` and `destroy` sections
+- External resources are deleted when your Okteto namespace is deleted
+- External resources are covered by Okteto's garbage collection process
```

Comment:
Should we link https://www.okteto.com/docs/administration/cleanup/ here?

Reply:
Latest commit adds that link 👍